src/app/ReviewService/ReviewPrompter.iOS.cs

```diff
@@ -11,14 +13,40 @@ namespace ReviewService;
 /// </summary>
 public sealed class ReviewPrompter : IReviewPrompter
 {
+	private readonly ILogger _logger;
+
+	public ReviewPrompter(ILogger<ReviewPrompter> logger)
+	{
+		_logger = logger ?? NullLogger<ReviewPrompter>.Instance;
+	}
+
 	/// <inheritdoc/>
 	public Task TryPrompt()
 	{
-		Foundation.NSRunLoop.Main.BeginInvokeOnMainThread(() =>
+		var tcs = new TaskCompletionSource();
+
+		Foundation.NSRunLoop.Main.BeginInvokeOnMainThread(async () =>
 		{
-			// TODO.
+			try
+			{
+				_logger.LogDebug("Trying to prompt the user for a review.");
+
+				await CrossStoreReview.Current.RequestReview(false);
+
+				_logger.LogInformation("Prompted the user for a review.");
+
+				tcs.SetResult();
+			}
+			catch (Exception e)
+			{
+				_logger.LogError(e, "Failed to prompt the user for a review.");
+				tcs.SetException(e);
+
+				throw;
```

Comment:
Don't re-throw the exception because that would crash the app. Passing the exception to the tcs is enough.